requirements/base.txt

```diff
@@ -7,6 +7,7 @@ python-dateutil==2.8.1
 django-messagegroups==0.4.5
 django-registration-redux==2.7
 django-sendfile==0.3.11
```

Comment:
```suggestion
```

The django-sendfile dependency can be removed then, right?

Reply:
True, I missed that. (Added a commit for that including exact version pinning as well)